Source/WebKit/Shared/JavaScriptEvaluationResult.cpp

```diff
@@ -69,4 +69,10 @@ JavaScriptEvaluationResult& JavaScriptEvaluationResult::operator=(JavaScriptEval
 
 JavaScriptEvaluationResult::~JavaScriptEvaluationResult() = default;
 
+WTF::TextStream& operator<<(WTF::TextStream& stream, JavaScriptEvaluationResult::NullType nullType)
+{
+    stream << (nullType == JavaScriptEvaluationResult::NullType::NullPointer ? "nil" : "NSNull");
```

Comment:
"NSNull" feels like the wrong log in this cross-platform code

Reply:
nvm you were right